src/test/java/seedu/address/logic/parser/lesson/EditLessonCommandParserTest.java

```diff
@@ -0,0 +1,28 @@
+package seedu.address.logic.parser.lesson;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.lesson.EditLessonCommand;
+
+public class EditLessonCommandParserTest {
+    private final EditLessonCommandParser parser = new EditLessonCommandParser();
+
+    @Test
+    public void parse_noFieldsProvided_failure() {
+        assertParseFailure(parser, "", String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+            EditLessonCommand.MESSAGE_USAGE));
+
+
+    }
```

Comment:
Can remove the 2 lines of whitespace here

Reply:
done!